### PR TITLE
release cargo-apk 0.8 and ndk-build 0.4

### PR DESCRIPTION
**Pull request by msiglreith**
Requested in https://github.com/rust-windowing/android-ndk-rs/pull/138#issuecomment-873394048

### CODE REVIEW

**Comment by MarijnS95** [comment]
@msiglreith It looks like this is missing the `version =` bumps themselves?

**Comment by msiglreith** [comment]
oh..🤦
thanks!

File: cargo-apk/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Unreleased
```
Comment (MarijnS95) [inline cargo-apk/CHANGELOG.md:1]: Might want to keep this around for the next release... I accidentally removed it in one of the previous releases too and it botched some rebases.

**Review by MarijnS95 — APPROVED** [review]

**Comment by MarijnS95** [comment]
Shouldn't this be a patch bump btw? We're still on `0.` and no breaking changes were made, last I remember.

**Comment by msiglreith** [comment]
thanks for reviews!
yes probably a patch bump would have been enough as well

**Pull request merged**